changelog.md

```diff
@@ -1,5 +1,10 @@
-### 2019-4-16 1.0.0-preview.3
-- Added support for proxy.
+### 2019-4-22 1.0.0-preview.3
+- Update the `isNode` utility to treat Electron apps as Node apps instead of Browser apps. [PR 40](https://github.com/Azure/amqp-common-js/pull/40)
```

Comment:
Update the `isNode` utility to return true when run under Electron.

Reply:
@bterlson I didn't like "under Electron". Changed to below instead :)

> Update the `isNode` utility to return `true` when run in Electron applications